tasks/stages/update-website

```diff
@@ -26,7 +26,7 @@ if [[ "x$SKIP_COMMIT" = "x" ]]; then
 
   # add all resources
   git add -A
-  git commit -m "chore(project): bump dmn-js to $TRAVIS_TAG"
+  git commit -m "chore: bump dmn-js to $TAG"
```

Comment:
Maybe we can already use our new conventional commit message:

```sh
deps: bump dmn-js to $TAG
```

Reply:
Makes sense. Thanks for the idea.

Reply:
Done.